client/src/util/quantile.js

```diff
@@ -0,0 +1,29 @@
+/*
+quantiles - calculate quantiles for the typed array.
+
+Currently interpolates to 'lower' value.
+
+Arguments:
+
+	* qs - array of quantiles to compute.
```

Comment:
Two things
qs -> quantArr
can you also specify in the comment that the array of quantiles should be floats between 0 and 1?

Reply:
fixed